test/testoauth.cpp

```diff
@@ -122,7 +122,7 @@ class OAuthTestCase : public QObject
     QScopedPointer<OAuth> oauth;
 
     virtual void test() {
-        fakeAm = new FakeAM({});
+        fakeAm = new FakeAM({}, nullptr);
```

Comment:
Why do you need to specify this explicitly? Can we make `nullptr` the default argument?

Reply:
I prefer to force the user